src/test/java/fr/lunatech/timekeeper/timeutils/TimeSheetUtilsTest.java

```diff
@@ -261,22 +260,22 @@ void shouldComputeLeftOver_NoDurationType() {
                 null,
                 4,
                 null,
-                generateTestEntries(1,8L),
+                generateTestEntries(1, 8),
                 START_DATE
         );
 
         //THEN: limit is consider by day, therefor 3 is returned
         assertThat(TimeSheetUtils.computeLeftOver(timesheet),is(3L));
     }
 
-    private List<TimeEntry> generateTestEntries (int numberOfEntry, long hourPerEntry){
+    private List<TimeEntry> generateTestEntries (int numberOfEntry, int hourPerEntry){
         LocalDateTime now = LocalDateTime.now();
-        LocalDateTime nowPlusSomeHours = now.plusHours(hourPerEntry);
+        if (hourPerEntry < 0) throw new IllegalStateException("Invalid duration");
         List<TimeEntry> entries = new ArrayList<>();
```

Comment:
Aesthetic: a loop for, could be replaced with a stream?

Reply:
Since it's in a unit test, I think we can keep it as it is, no?